bundles/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/AutelisBindingProvider.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.autelis;
+
+import org.openhab.core.binding.BindingProvider;
+import org.openhab.core.items.Item;
+
+/**
+ * 
+ * Provides a mapping between openHAB {@link Item}s and a Autelis Pool controller.
+ * @author Dan Cunningham
+ * @since 1.7.0
+ */
+public interface AutelisBindingProvider extends BindingProvider {
+
+	/**
+	 * Returns the name of the item
+	 * @param itemName
+	 * @return
+	 */
+	public Item getItem(String itemName);
+	
+	/**
+	 * return the binding config for an item 
+	 * @param itemName
+	 * @return
+	 */
+	public String getAutelisBindingString(String itemName);
```

Comment:
i would suggest to rename to getAutelisBindingConfigString or better directly return the BindingConfiguration object here